### DISCUSSION TITLE
Please publish a Go + Protobuf release

### DISCUSSION THREAD

**Discussion by jbcpollak** [foxglove/community#995] [General]
Summary says it all - would be great if you could publish a Go + Protobuf library

**Comment by linear** [comment]
<p><a href="https://linear.app/foxglove/issue/FG-7469/publish-go-protobuf">FG-7469 Publish go + protobuf</a></p>

**Comment by jbcpollak** [comment]
Added a proof of concept PR here - https://github.com/foxglove/schemas/pull/145
